packages/@react-spectrum/provider/src/Provider.tsx

```diff
@@ -185,6 +185,9 @@ const ProviderWrapper = React.forwardRef(function ProviderWrapper(props: Provide
   );
 });
 
+/**
+ * Returns the various settings and styles applied by the immediate parent Provider.
```

Comment:
If the immediate parent provider wasn't provided with some value, this could return that value from the 2nd parent provider right? Do we need to be more clear about that here, or is kind of implied?

Reply:
We should probably get rid of the immediate part. It's the aggregate through the tree since we use previous provider context to make the next one

Reply:
same down below, it's whatever the nearest provider is that actually set one of those values

Reply:
Something like this might work, although the second sentence might just add confusion.
```js
/**
 * Returns the various settings and styles applied by the parent Provider. A Provider will use values from its parent Provider (if one exists) for any values not provided to itself.
*/
```

Reply:
Updated the description to "nearest parent Provider" and added clarification to `useProvider` doctype explaining that the nearest provider's explicitly set props override preceeding providers. Lemme know if you think it might be a bit too confusing and if I should just link to the docs

Reply:
annnnd I just refreshed and saw your suggestion @reidbarber. Happy to use that if you think it might be easier to understand, I'm not great at judging that haha

Reply:
I look yours better, less confusing wording haha